### ISSUE TITLE
Black screen at opening, cant even see the Konami logo cant play

### ISSUE THREAD

**Issue by ulumete** [elishacloud/Silent-Hill-2-Enhancements#359]
I have been trying to play the game all day. First I tried to play base game without this enhanced mod, it did not work. Then I stared the sh2pc with windows 98 service pack so it used only one core, that worked but it was useless anyways cuz all the lagging made the game unplayable. Then I downloaded this enhanced mod with all the instructions more than once, but I cant get pass the black screen, cant see the Konami logo. Whatever I have done did not work and I am going to get crazy. Problem seems to be because of singlecoreaffinity thing. When I read the d3d8 log the last line is always "8840 10:16:22.432 Setting SingleCoreAffinity...". I checked it by task manager and it was in fact running by only one core. I tried giving it different numbers but it did not work, pleaseee help.

**Comment by Polymega** [comment]
Hello,
Please upload a copy of your `d3d8.log` file. Drag and drop the file into the text box here and it should upload it for you. Thanks

**Comment by ulumete** [comment]
[d3d8.log](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/5731501/d3d8.log)
here you go mate

**Comment by Polymega** [comment]
Thanks. Let's try a few tests first to see if we can get you up and running. 

To start, right click `sh2pc.exe` and go to **Properties**. Make sure all of the **Compatibility** settings are **unchecked**.

![image](https://user-images.githubusercontent.com/27395260/102922789-973d7780-445c-11eb-9a44-7f55d5cf52e9.png)

Next, I see some files that [shouldn't be in the game's directory](http://enhanced.townofsilenthill.com/SH2/install.htm#directoryTree). **Remove** the following files from the game's directory:

```
Command Prompt.lnk
device.7z
msvcp140.dll
vcruntime140.dll
```

Let's try that for now to see if it makes the game run for you. Let me know.

**Comment by ulumete** [comment]
everything is unchecked, command prompt and device.7z are useless anyways I can delete those. The other two are needed, otherwise I get dll errors. I deleted the first two and still the same thing.

**Comment by Polymega** [comment]
> The other two are needed, otherwise I get dll errors. I deleted the first two and still the same thing.

I'm thinking this may be related to your issue of not getting the game to run. As you also mentioned you couldn't get the vanilla/base game to run by itself, either. It sounds like you're missing some files in your Windows that's needed to get the game to run. (My guess is a version of Microsoft Visual C++? Not sure...)

Do you have issues running other games? In particular, older games?

**Comment by ulumete** [comment]
no, I do not think so. Like I said when launched the vanilla game with windows 98 service pack, it opened fine. I also downloaded the directX thing from the download instructions. I can also play, for example older total war titles.

**Comment by Polymega** [comment]
Hmm... This is a complete shot in the dark, but make a **copy** of your `sh2pc.exe` file and rename it to something different like `sh2pc_ee.exe` and see if the game launches when you run it through that executable.

**Comment by ulumete** [comment]
[d3d8.log](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/5731673/d3d8.log)
it didnt work

**Comment by Polymega** [comment]
Sorry you're having issues. We may need to get @elishacloud's thoughts on this. He might know to try something I can't think of.

**Comment by ulumete** [comment]
ok but it doesnt matter anyways :/ So I check here later. Who knows. maybe I get to play this damn game after all.

**Comment by elishacloud** [comment]
> everything is unchecked, command prompt and device.7z are useless anyways I can delete those. The other two are needed, otherwise I get dll errors. I deleted the first two and still the same thing.

The `msvcp140.dll` and `vcruntime140.dll` files are part of the [Microsoft Visual C++ 2015 Redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=53840) package.  However, nothing in the Silent Hill 2 base game uses the Microsoft Visual C++ 2015 Redistributable.  Is it possible you have a bad copy of Silent Hill 2?

You could try renaming `dsound.dll` to see if this is using Microsoft Visual C++ 2015 Redistributable (I cannot test it out right now).  Also, rather than adding these files to the games folder, it would be better to delete these files and actually install the Redistributable from here:  https://www.microsoft.com/en-us/download/details.aspx?id=53840

> First I tried to play base game without this enhanced mod, it did not work.

If you cannot get the game to work without the enhancement mod then it will not work with the enhancement mod.  This mod is not going to cause a non-working game to start working.

Also, looking at your logs, I see this:

```txt
5100 11:26:51.644 Environment variable __COMPAT_LAYER = "DetectorsAppHealth DetectorsXP"
```

It is hard to find anything about these compatibility layers, but they are not normally set.  `DetectorsXP` might be ok, but, every time I have seen `DetectorsAppHealth` it causes an issue.  It is speculated that `DetectorsAppHealth` is caused by Windows Defender.

See here:
* https://github.com/commercialhaskell/stack/issues/3765#issuecomment-570785973
* https://github.com/rikvdkleij/intellij-haskell/issues/450#issuecomment-570779968

I would recommend getting the game to run correctly without any of the enhancement mod components before enabling the enhancement mod.

**Comment by Polymega** [comment]
Closing this ticket. Feel free to re-open it if you are still experiencing problems.
